resources/warnings.json

```
@@ -9130,7 +9130,7 @@
     "versions": [
       {
         "lastVersion": "2.10.2",
-        "pattern": ".*"
+        "pattern": "(1|2[.]\\d[0]?)(|[.-].+)"
```

The current regex will false identify future version 2.20 as affected.

```suggestion
        "pattern": "(1|2[.][0-9]|2[.]10[.]1)(|[.-].+)|(2[.]10)"
```

---

Good catch!